### DISCUSSION TITLE
Media Play Scrubbing

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#809] [Q&A] [answered]
Was there a discussion at one point to have the ability to scrub to a specific portion of a media file and play from that point? Or have I missed something?
I had a panic phone call this morning from an operator who asked if instead of editing a PT that they just move to the part of the video they wanted to play (to exclude chairman introduction) (this is a remote area congregation). I thought there was provision to do that but I cannot find it.
If I had known I would have quickly edited it for them and uploaded ready to go but as it was I was rung 5 minutes before the meeting. (Isn't it always the way?!) 😂

**Comment by mtdvlpr** [comment, marked as answer]
There are two ways. You can press the time button on the video preview to set a custom start and end time or you can pause the video and then scrub the video to the part you want and press play.

**Reply by Chipperchap** [reply]
Wow that is amazing!  I never knew that (and here I was thinking I was pretty aware of all things M<sup>3</sup>!)
Well done and thank you. 

By the way, where is the donate button? 💳

**Reply by sircharlo** [reply]
[donate.jw.org](https://donate.jw.org) is the way to go 🙂
